lib/docs/scrapers/enzyme.rb

```diff
@@ -0,0 +1,24 @@
+module Docs
+  class Enzyme < UrlScraper
+    self.type = 'simple'
+    self.release = '3.10.0'
+    self.base_url = 'https://airbnb.io/enzyme/'
+    self.root_path = 'index.html'
+    self.links = {
+      code: 'https://github.com/airbnb/enzyme'
+    }
+
+    html_filters.push 'enzyme/entries', 'enzyme/clean_html'
+
+    options[:skip] = %w(CHANGELOG.html docs/future.html CONTRIBUTING.html)
```

Comment:
Can you also ignore `docs/guides.html` and `docs/api/index.html`? It’s pretty much a duplicate of the regular Guides type page.

```suggestion
    options[:skip] = %w(CHANGELOG.html docs/future.html docs/guides.html docs/api/index.html CONTRIBUTING.html)
```

I wish there was a way to specify custom type pages…